addons/calendar/views/calendar_views.xml

```diff
@@ -309,6 +309,20 @@
         </field>
     </record>
 
+    <record id="view_users_form" model="ir.ui.view">
+            <field name="name">res.users.form</field>
+            <field name="model">res.users</field>
```

Comment:
According to guidelines, should be:
ID: `id="res_users_view_form"`
Name: `<field name="name">res.users.view.form.inherit.calendar</field>`

I know the ones you're copying from do not respect guidelines themselves but we try to respect them as much as possible when introducing new views and new overrides.